workflows/post_process_run/tests/test_gsutil.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+import pytest
+
+from subprocess import CalledProcessError
+from unittest.mock import MagicMock
+from fv3post.gsutil import upload_dir, GSUtilResumableUploadException
+
+
+UPLOAD_EXCEPTION_STR = CalledProcessError(-1, None, "ResumableUploadException")
+UPLOAD_EXCEPTION_BYTES = CalledProcessError(-1, None, b"ResumableUploadException")
+OTHER_EXCEPTION = CalledProcessError(-1, None, output=b"Other")
+
+
+@pytest.mark.parametrize(
+    ("side_effect", "exception", "call_count"),
+    [
+        (UPLOAD_EXCEPTION_STR, GSUtilResumableUploadException, 3),
+        (UPLOAD_EXCEPTION_BYTES, GSUtilResumableUploadException, 3),
+        (OTHER_EXCEPTION, CalledProcessError, 1),
+        (None, None, 1),
+        ([UPLOAD_EXCEPTION_STR, None], None, 2),
+        ([UPLOAD_EXCEPTION_BYTES, None], None, 2),
+    ],
+)
+def test_upload_dir(monkeypatch, side_effect, exception, call_count):
```

Comment:
The tests will currently pass if there's a bug in the code where the code checks that the message is equal to `"ResumableUploadException"`, instead of that it contains the exception. Can you add one case where the error message is a longer string containing the exception somewhere in the middle?

Reply:
I modified the existing test cases such that they test this.